### DISCUSSION TITLE
different rendering during test and during review

### DISCUSSION THREAD

**Discussion by linediconsine** [cypress-io/cypress#16517] [Component Testing]
Hi All,
I am testing our website based on boostrap and I see different Layout from test running to then see the page preview on the test runner.

here is and example : 

Page:
![Schermata 2021-05-14 alle 14 39 57](https://user-images.githubusercontent.com/1525362/118273219-fd866d80-b4c3-11eb-96a1-c3fb9e9ee4e0.png)

White the test run I see the page corrected by the if I move the mouse on the step,  I see : 
![Schermata 2021-05-14 alle 14 40 18](https://user-images.githubusercontent.com/1525362/118273332-17c04b80-b4c4-11eb-903b-a7cc5ce4b8af.png)

My cypress.json is : 
```
{
  "projectId": "hffses",
  "viewportWidth": 1920,
  "viewportHeight": 1080,

  "experimentalStudio": true
}
```

I tried to disable experimental but this didnt change the result

Thanks in advance for any help on this

Marco

**Comment by lmiller1990** [comment]
Are you using the component testing runner (`cypress open-ct`)? The category is  component testing, but it looks in you are in the e2e runner (`cypress open`).

Going to need more information to go on - it looks like some CSS is not loaded, but it's impossible to help more without your actual repository. You could check the network tab - is any CSS or style sheets not getting loaded?

**Comment by jennifer-shehane** [comment]
In some cases the DOM snapshot when reviewing the history may not be correctly regenerated, this would be a bug. If you can provide a way for us to reproduce the issue fully, open an issue and we can review the situation.
